dGame/dComponents/RebuildComponent.h

```diff
@@ -19,7 +20,7 @@ class Entity;
  */
 class RebuildComponent : public Component {
 public:
-	static const uint32_t ComponentType = COMPONENT_TYPE_REBUILD;
+	static const eReplicaComponentType ComponentType = eReplicaComponentType::REBUILD;
```

Comment:
Rebuild or quickbuild?  I've seen both in this PR and getting this translation correct is critical

Reply:
DLU rebuild == actual quick_build